src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -207,49 +220,69 @@ const AuthenticatorButton: UISchemaElementComponent<{
             id={`${iconName}-usageDescription`}
             textAlign="start"
             sx={{
-              fontSize: '.875rem',
+              fontSize: tokens.TypographySizeSubordinate,
+              color: tokens.TypographyColorSubordinate,
               margin: 0,
-              marginBlockEnd: '6px',
-              color: 'text.secondary',
+              marginBlockStart: '6px',
             }}
             data-se="authenticator-button-usage-text"
             className="authenticator-usage-text"
           >
             {usageDescription}
           </Typography>
         )}
+        {isEnroll && (
+          <Box
+            className="cta-button authenticator-button"
```

Comment:
Even though we aren't updating Authenticator List components to Odyssey components, we should still try to remove all uses of `className`. It seems like this might be a big ask and this PR already has a lot of file changes, so I'm open to making a separate ticket for going through these components and removing `className` as well as updating relevant spec tests. Lmk what you think

Reply:
I'm fine with removing classnames in this PR.